doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+
+  * nRF9160 GPS - (? where is that? what does it do?)
+
+
+Updated samples and applications
+--------------------------------
+
+* :ref:`asset_tracker`:
+
+  * Updated to use the generic cloud API.
+  * Added a user interface module to the application to facilitate the use of buttons, LEDs, buzzer, and NMOS transistors.
+  * Added a cloud command decoder module that parses incoming JSON strings.
+  * Added an application reboot in the case that MQTT CONNACK is missing from the nRF Cloud server.
+  * Fixed a bug where invalid RSRP values (not known / not detectable) were sent to the cloud.
+  * Added service information JSON to the device information shadow data.
+  * Added light sensor handling.
+  * Added FOTA support for application and modem.
+
+* :ref:`aws_fota_sample`:
+
+  * Added a warning message when provisioning certificates stating that the certificates are stored in application core flash (readable flash) and are visible on modem traces.
+  * Changed the default security tag to not be the same as nRF Cloud's security tag, to ensure that users do not overwrite their nRF Cloud certificates.
+  * Created a separate nRF Cloud configuration option for the sample.
+  * Added device shadow update to the sample.
+
+Updated libraries
+-----------------
+
+* :ref:`modem_info_readme`:
+
+  * Reworked the architecture to support a parameter storage module.
+  * Increased the available data.
+  * Extended the modem information with IMEI, IMSI, date, and time information.
+  * Changed the modem information to be handled as JSON object instead of strings.
+  * Fixed known bugs.
+
+* :ref:`lib_fota_download`:
+
+  * Add support for performing MCUboot upgrades.
+    To support this, ``CONFIG_SECURE_BOOT=y`` must be set.
+    To send an upgrade, provide the path to both the S0 and the S1 candidate (separated by a space) to the file parameter of :cpp:func:`fota_download_start`.
+    Both candidates are generated by the build system if ``CONFIG_MCUBOOT_BUILD_S1_VARIANT=y`` is set.
+
+* :ref:`lib_aws_fota` and :ref:`lib_aws_jobs`:
+
+  * Implemented fetching of AWS jobs.
+    This allows a device to be updated if it is offline at the time the update is created.
+  * Refactored the code (improved function names, extracted common functionality, re-used topic buffers given to AWS jobs).
+  * Added unit tests for AWS jobs.
+  * Removed device shadow update from the library.
+
+* MQTT library:
+
+  * Dropped the nRF Connect SDK copy of the MQTT library and adopted Zephyr's :ref:`zephyr:mqtt_socket_interface` library instead.
+
+* :ref:`lib_spm`:
+
+  * Added a new non-secure-callable function :cpp:func:`spm_firmware_info`.
+
+* :ref:`lib_nrf_cloud`:
+
+  * Adopted to the new device shadow format.
+
+* at_host (``lib/at_host``):
+
+  * Updated to use a dedicated workqueue instead of the system workqueue.
+  * Miscellaneous fixes and improvements.
+
+* :ref:`at_cmd_parser_readme`:
+
+  * Refactored the library.
+  * Added :ref:`at_notif_readme` dispatcher.
```

Comment:
`at_notif` is a new library.

Reply:
Moved up to the section for added libraries:
  * :ref:`at_notif_readme` - dispatches AT command notifications to registered modules.